tcms/testruns/tests/test_views.py

```diff
@@ -119,7 +119,7 @@ def test_show_create_new_run_page(self):
         })
 
         # Assert listed cases
-        for i, case in enumerate((self.case_1, self.case_2, self.case_3), 1):
+        for case in enumerate((self.case_1, self.case_2, self.case_3), 1):
```

Comment:
same thing as above. read the docs for the `enumerate()` function and see what type of values it returns.